ssl/ssl_ciph.c

```diff
@@ -2168,3 +2168,21 @@ int ssl_cert_is_disabled(size_t idx)
         return 1;
     return 0;
 }
+
+/* 
+ * Default list of TLSv1.2 (and earlier) ciphers
+ * SSL_DEFAULT_CIPHER_LIST deprecated in 3.0.0
+ */
+const char *OSSL_default_cipher_list(void)
+{
+    return SSL_DEFAULT_CIPHER_LIST;
```

Comment:
Isn't this going to fail to compile if the user requests no-deprecated?

Reply:
Damn, was hoping I could use the same definition.

Reply:
Hmm... it does not fail to compile with `no-deprecated`, since that deals with deprecated features (such as ssl2 and dso). Even with `--api=3.0.0` it still compiles. Once we move beyond 3.x, it will become an issue.

Unless I'm missing something?